test/chainsaw/testmetrics/metrics-keda/chainsaw-test.yaml

```diff
@@ -33,6 +33,16 @@ spec:
       catch:
         - podLogs:
             selector: app=podtato-head
+        - describe:
```

Comment:
I think this is the same as podLogs in the line above so it is redundant

Reply:
podLogs provide logs of the pod, describe is a different command in my opinion

Reply:
yeah I realised and resolved the conversation 🤔